packages/fxa-payments-server/src/components/CouponForm/index.tsx

```diff
@@ -75,78 +75,76 @@ export const CouponForm = ({ planId, coupon, setCoupon }: CouponFormProps) => {
   };
 
   return (
-    <section
+    <div
```

Comment:
huh does this make a difference wrt styles?

Reply:
not that i can tell? none of our css was selecting on element type, just classes. i imagine there are some inherent differences (obivously) but i think we are pretty explicit with our css and it appears to be rendering correctly on brave (chromium based) and firefox as well across different screen sizes on both.

Reply:
I was curious why you made that change if it did not make a difference.